pkgs/by-name/ap/apostrophe/package.nix

```diff
@@ -1,59 +1,117 @@
-{ lib, stdenv, fetchFromGitLab, meson, ninja
-, wrapGAppsHook3, pkg-config, desktop-file-utils
-, appstream-glib, python3Packages, glib, gobject-introspection
-, gtk3, webkitgtk, glib-networking, gnome, gspell, texliveMedium
-, shared-mime-info, libhandy, fira, sassc
+{
+  lib,
+  fetchFromGitLab,
+  gtksourceview5,
+  libspelling,
+  fetchFromGitHub,
+  python3Packages,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  desktop-file-utils,
+  appstream-glib,
```

Comment:
Packages are generally moving away from appstream-glib as it is unmaintained. Is it really needed?